app/server/app.js

```diff
@@ -72,128 +70,138 @@ app.get('/live-affluence', function(req, res){
     
 });
 
-// endpoint receiving the sms from twilio
-app.post('/twilio', function(req, res) {
+// // endpoint receiving the sms from twilio
+// app.post('/twilio', function(req, res) {
```

Comment:
DO NOT PUSH THIS TO PROD!
Sensors as they are expect to be able to send to 6element.ants.builders, which in our case has become the app container.
The app container needs the /twilio endpoint, otherwise nothing receives the sensor data